projects/packages/jetpack-mu-wpcom/src/features/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-launchpad.php

```diff
@@ -33,9 +33,19 @@ public function register_routes() {
 			$this->namespace,
 			$this->rest_base,
 			array(
-				'methods'             => WP_REST_Server::READABLE,
-				'callback'            => array( $this, 'get_data' ),
-				'permission_callback' => array( $this, 'can_read' ),
+				array(
+					'methods'             => WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_data' ),
+					'permission_callback' => array( $this, 'can_read' ),
+				),
+				array(
+					'methods'             => WP_REST_Server::EDITABLE,
+					'callback'            => array( $this, 'update_site_options' ),
+					'permission_callback' => array( $this, 'can_read' ),
+					'request_format'      => array(
+						'launchpad_checklist_tasks_statuses'      => '(array) Array of launchpad checklist tasks completion status',
```

Comment:
I'd like to see this simplified to just accept the same arg we're using in the GET endpoint, `checklist_statuses` - we can make the REST API interface a bit nicer than our clunky name :)